.github/workflows/run-dev.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         run: python -m permanent_upload dev files/
   notify:
     runs-on: ubuntu-latest
-    if: ${{ github.event.repository_dispatch && always() }}
+    if: ${{ github.event_name == 'repository_dispatch' && always() }}
```

Comment:
I wanted to ask you about this last week, but felt it was more important to get #13 merged:

Why have both? Do we need to filter on the event type? Why not have it set to always? If we do need to filter, is there some benefit to also having the `always()`?

Reply:
Yes, we need to filter on event type because it's now running as part of CI. We need `always()` because we want it to run even if `execute` fails.

Reply:
> Yes, we need to filter on event type because it's now running as part of CI.

Aha, got it - we're running this workflow on pull requests, and don't want to report success / failure to slack, since GitHub already presents the results in the PR checks. Thanks!

> We need always() because we want it to run even if execute fails.

Aha, that makes sense - thank you!